src/Hng.Application/Dto/ResponseDto.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Hng.Application.Dto
+{
+    public record SuccessResponseDto(string message, int status_code, bool success = true);
```

Comment:
Bros. Don't use records. Use normal classes. Records adds a bunch of what we don't need. Keep it simple brother. 

You should have only one generic class for `ResponseDto`. Something like
```csharp
public class ResponseDto<T> {
   T Data {get; set;} // where T is the type of data being returned
   int StatusCode {get; set;}
   bool Success {get; set;}
   string Message {get; set;}
}
```
Use `JsonProperty` attributes to format the property names the way they should appear on the API spec